Rubberduck.Core/UI/Inspections/InspectionResultsControl.xaml

```diff
@@ -236,8 +239,8 @@
                                 <Setter Property="Command" Value="{Binding Command}" />
                                 <Setter Property="CommandParameter" Value="{Binding Fix}" />
                                 <Setter Property="Header" Value="{Binding Description}" />
-                                <Setter Property="Background" Value="Transparent"></Setter>
-                            </Style>
+								<Setter Property="Background" Value="Transparent" />
+							</Style>
```

Comment:
Weird whitespace craze...

Reply:
As mentioned above, probably tabs instead of spaces.

Reply:
This is definetly due to tabs instead of spaces. Please replace the tabs with spaces.